### ISSUE TITLE
commands not working

### ISSUE THREAD

**Issue by Tentacless** [budanthara/LINE-ChatBot#4]
I'm probably doing something wrong but when I type "python linebot.py --execute" it just shows a list of the available commands. The only ones that work are -sg -sr -sc. Also I always get this message but I ignore it "WARNING:root:pafy: youtube-dl not found; falling back to internal backend. This is not as well maintained as the youtube-dl backend. To hide this message, set the environmental variable PAFY_BACKEND to "internal"."

Also, running in root gives me this
"[+] Please install -> No module named thrift.transport first!
[+] Login Failed! [+]
[+] Reason:  name 'LineClient' is not defined
Traceback (most recent call last):
  File "linebot.py", line 68, in <module>
    main()
  File "/home/user/LINE-ChatBot/src/bot.py", line 231, in main
    parser = argparse.ArgumentParser()
NameError: global name 'argparse' is not defined "

Basically I'm not entirely sure how to run the bot. At least I got the authToken.
